src/modules/markCompleted.js

```diff
@@ -0,0 +1,23 @@
+import { TasksClass } from './tasksClass.js';
+
+// mark task complete
+export const markComplete = () => {
+  //  get the elements
+  const taskBoxEls = document.querySelectorAll('.task-select-input');
+  const taskStatus = new TasksClass();
+  // get index and checkbox value by iterating
+  taskBoxEls.forEach((theEl, theElIndex) => {
+    // functions to execute when a task checkbox changes
+    const checkTaskElements = () => {
+      const taskBoxChecked = theEl.checked;
+      // const filteredTaskBoxEls = theEl.filter((a) => a.checked === true);
+      if (taskBoxChecked === true) {
+        taskStatus.taskCompleted(theElIndex, taskBoxChecked);
+      }
+    };
+    // events
+    theEl.addEventListener('change', checkTaskElements, false);
+  });
+};
```

Comment:
Nice job updating the task array whenever a checkbox is clicked. However, the project requirements state that all changes made should be saved to local storage. While your application saves completed status changes to local storage, the UI does not persist when the page is refreshed. This leads to a bug where a task has its completed status in local storage set to true but the checkbox is not checked. Please update your app so that changes in the UI are persisted on page refresh🙏

Reply:
Okay. Thanks for the clarification

Reply:
This is fixed. Thanks